designsafe/apps/rapid/templates/designsafe/apps/rapid/index.html

```diff
@@ -7,25 +7,8 @@
 
 
 {% block main %}
-<div class="rapid-header">
-  <div class="rapid-logo">
-    <img src="/static/scripts/rapid/images/logo.png" alt="Recon Portal logo"></img>
-  </div>
-  <div class="rapid-header-text">
-    <h3>
-      The Recon Portal is an interactive world map of natural hazard events with associated datasets. View available datasets by clicking on a marker, browsing the listing of events, or searching by clicking on "Show filter options".
-      <br>
-      <a href="/user-guide/tools/recon/"> <i role="none" class="fa fa-question-circle"></i> <b>Learn how to contribute your datasets.</b></a>
-    </h3>
-  </div>
-</div>
-<br>
-<br>
-<div id="rapid">
-  <div>
-    <div ui-view></div>
-  </div>
-</div>
+ 
+<div id="rapid-root" ></div>
```

Comment:
recon-portal-root?

should we change the django app from rapid to recon-portal?